milestone-delivery/deliveries/dappworld_learning_ecosystem-milestone-1.md

```diff
@@ -0,0 +1,14 @@
+# Milestone Delivery :mailbox:
+
+
+**The Ethereum address has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](../).**  
+
+* **PR Link:**  [DApp World Learning ecosystem](https://github.com/vechain/grant-program/blob/master/applications/dappworld_learning_ecosystem.md). 
+* **Milestone Number:** 1 
+
+VeChain introducton Course is completed and hosted on DApp World as per promised first milestone.
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Introduction to VeChain | https://dapp-world.com/course/vechain-dapp-development-UpGw  | ✓ The Course is Successfully completed along with project directory uploaded on DApp World github. Multiple tutorials are also uploaded in [Smartbooks](https://dapp-world.com/smartbook/interact-with-smart-contract-on-vechain-from-javascript-Q3oD) | 
+| 2.  | ... |...| ...| 
```

Comment:
If there is no new lines, please remove this line.

Reply:
Removed the line in merged application file